www/html/api-laravel/app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -12,6 +12,6 @@ class VerifyCsrfToken extends Middleware
      * @var array<int, string>
      */
     protected $except = [
-        //
+       'http://127.0.0.1:8000/api/clients'
```

Comment:
Aqui você pode colocar só o nome da rota, parametro definido em routes.php atraves do metodo name()

Route::post('/clients', NewClientController::class)->name('rotax');

```suggestion
       'rota-x'
```